src/shared/cpu-set-util.h

```diff
@@ -73,6 +74,7 @@ static inline void numa_policy_reset(NUMAPolicy *p) {
 }
 
 int apply_numa_policy(const NUMAPolicy *policy);
+int numa_to_cpu_set(const NUMAPolicy *policy, CPUSet *set);
 
 const char* mpol_to_string(int i) _const_;
 int mpol_from_string(const char *s) _pure_;
```

Comment:
the numa related stuff should maybe moved to numa-util.[ch]?

Reply:
I don't think we need a new module for NUMA stuff just yet, but then again,  should you block the merge I will do it.

Reply:
not gonna block it, but you#d earn extra point, for splitting up things? keeping code in managable units is always a good thing...

Reply:
(but totally happy if happens in separate commit/PR. and if you don't do it I would do that sooner or later myself...)